its/projects/CshtmlClassLib/CshtmlClassLib.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
maybe use .net 7 (As we said that is what we want to support for sure).
Same for the other project.